src/apis/answers.ts

```diff
@@ -0,0 +1,57 @@
+import { normalize } from "normalizr";
+import { client } from ".";
+import { aEntity } from "./schemas";
+
+// defining normalize return type manually as normalize has bad typescript support
+// general types used here are defined in ./src/types.d.ts file
+type Normalized<T = Result> = {
+  entities: {
+    answers: Record<number, Answer>;
+    users: Record<string, User>;
+  };
+  result: T;
+};
+
+type fetchQAArgs = {qId: number, page: number};
+async function fetchQuestionAnswers({qId, page}: fetchQAArgs) {
+  const { data } = await client.get(`/questions/${qId}/answers`, {
+    params: { page }
+  });
```

Comment:
Inject the page directly as you are already using string literals
```suggestion
  const { data } = await client.get(`/questions/${qId}/answers?page=${page}`)
```